src/component/mod.rs

```diff
@@ -239,4 +253,9 @@ impl ComponentHandle {
             component_stats,
         }
     }
+
+    // The heartbeat function is called periodically
+    pub fn heartbeat(&mut self) {
```

Comment:
This might be github making it hard to tell, but where is this being defined/why do we need a wrapper for the function?

Reply:
I just didn’t want to poke through a layer of abstraction here